src/Compilers/CSharp/Portable/Binder/Binder_Statements.cs

```diff
@@ -3202,14 +3206,16 @@ internal virtual ImmutableArray<LabelSymbol> Labels
                 methodName,
                 arity: 0,
                 basesBeingResolved: null,
-                options: LookupOptions.Default,
+                options: lookupOptions,
                 originalBinder: this,
                 diagnose: false,
                 useSiteDiagnostics: ref useSiteDiagnostics);
 
             diagnostics.Add(syntaxExpr, useSiteDiagnostics);
 
-            if (!lookupResult.IsMultiViable)
+            MethodSymbol patternMethod = null;
```

Comment:
>r [](start = 30, length = 1)

This declaration can be moved to the initialization on line 3252.  It has no usage before then.